quinn-proto/src/connection/mod.rs

```diff
@@ -2442,6 +2450,7 @@ where
             sending_ecn: self.path.sending_ecn || !maybe_rebinding,
             challenge: Some(self.rng.gen()),
             challenge_pending: true,
+            pacing: pacing::Pacer::new(0),
```

Comment:
Why does this make sense? Why not use the current congestion window from the congestion controller we're setting for the path?